libs/community/langchain_community/chains/pebblo_retrieval/base.py

```diff
@@ -326,6 +327,7 @@ def _get_app_details(  # type: ignore
             framework=framework,
             chains=chains,
             plugin_version=PLUGIN_VERSION,
+            langchain_version=langchain_community.__version__,
```

Comment:
Not sure if you care, but this is the community version. You probably want the core version as well .

langchain-community
langchain
langchain-core

are all different packages

Reply:
Yes, @eyurtsev updated this